.travis.yml

```diff
@@ -33,13 +30,15 @@ install:
 
 script:
   - ./gradlew check
-  - ./gradlew clean install --stacktrace
-  - ./gradlew clean build --stacktrace
+  - ./gradlew clean install -x groovyDoc -x codenarcMain -x codenarcTest --stacktrace
+  - ./gradlew clean build -x groovyDoc -x codenarcMain -x codenarcTest --stacktrace
```

Comment:
This avoids running unnecessary and long-running tasks as they have already been run in `./gradlew check`.